packages/chevrotain/test/utils/matchers.ts

```diff
@@ -6,10 +6,24 @@ export function setEquality(actual: any[], expected: any[]): void {
     expect(expected).to.have.lengthOf(actual.length)
 }
 
-export function createRegularToken(tokType, image = ""): IToken {
+export function createRegularToken(
+    tokType,
+    image = "",
+    startOffset = 1,
+    startLine = 1,
+    startColumn = 1,
+    endOffset = Number.MAX_VALUE,
```

Comment:
Not sure this matters what value we use but why are we using MAX_VALUE here?

Reply:
Indeed, I just came back from vacations a couple of days ago, I have to remember exactly what/why I changed :sweat_smile: